sycl/CMakeLists.txt

```diff
@@ -126,3 +132,51 @@ add_custom_target( sycl-toolchain
 add_subdirectory( test )
 add_subdirectory( unittests )
 add_subdirectory( tools )
+
+# Package deploy support
+# Listed here are component names contributing the package
+set( SYCL_TOOLCHAIN_DEPLOY_COMPONENTS
+     clang
+     clang-offload-wrapper
+     clang-offload-bundler
+     llc
+     llvm-as
+     llvm-ar
+     llvm-dis
+     llvm-no-spir-kernel
+     llvm-spirv
+     llvm-link
+     llvm-objcopy
+     opt
+     clang-resource-headers
+     opencl-headers
+     sycl-headers
+)
+
+# Use it as fake dependency in order to force another command(s) to execute.
+add_custom_command(OUTPUT __force_it
+    COMMAND "${CMAKE_COMMAND}" -E echo
+)
+#Serialize installation to avoid missing components due to build race conditions
+set(__chain_dep __force_it)
+
+set(manifest_list)
+foreach( comp ${SYCL_TOOLCHAIN_DEPLOY_COMPONENTS} )
+    message( STATUS "Adding component ${comp} to deploy")
+
+    set (manifest_file ${CMAKE_CURRENT_BINARY_DIR}/install_manifest_${comp}.txt)
+        add_custom_command(OUTPUT ${manifest_file}
```

Comment:
Indentation if off.

Reply:
Valery is on vacation. I fixed this indentation issue and re-based the patch 
to get passes on those 2 failed LIT tests on Windows (boolean.cpp and vector_extracts.cpp) and to check that the nothing got broken while this patch was in review.